lib/redis/ick.rb

```diff
@@ -476,7 +476,8 @@ def _eval(lua,ick_key,*args)
       local ick_cset_size = redis.call('ZCARD',ick_cset_key)
       local ick_stats     = {
         'key',        ick_key,
-        'keys',       { ick_key, ick_pset_key, ick_cset_key },
+        'pset_key',   ick_pset_key,
+        'cset_key',   ick_cset_key,
```

Comment:
This nested array was what aggravates Twemproxy and causes https://github.com/ProsperWorks/redis-ick/issues/3 and https://github.com/twitter/twemproxy/issues/537.